src/index.html

```diff
@@ -4,8 +4,8 @@
     <title>Firefox Marketplace</title>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width">
-
-    <link rel="stylesheet" type="text/css" href="/media/css/bundle.css"/>
+    <link rel="stylesheet" href="//code.cdn.mozilla.net/fonts/fira.css">
```

Comment:
I did this because we're serving an out-of-date version of Fira, and have problems serving some special characters. Keeping our fonts up-to-date seems...problematic.

I can be talked into switching out of this, but pulling from the CDN makes a lot of sense. If you want to be able to develop locally, just install the fonts locally and let this 404?